tools/check-rule-examples.js

```diff
@@ -137,17 +167,8 @@ async function findProblems(filename) {
                     }
                     const { value } = commentParser.parseDirective(comment.value);
                     const parseResult = commentParser.parseJSONLikeConfig(value);
-                    const parseError = parseResult.error;
 
-                    if (parseError) {
-                        problems.push({
-                            fatal: true,
-                            severity: 2,
-                            message: parseError.message,
-                            line: comment.loc.start.line + codeBlockToken.map[0] + 1,
-                            column: comment.loc.start.column + 1
-                        });
```

Comment:
These problems will be reported by `Linter#verify()` so there's no need for a duplicate check here.